File: simplex.java/src/main/java/kr/co/tmon/simplex/exceptions/UnclonableTypeException.java
package kr.co.tmon.simplex.exceptions;

public class UnclonableTypeException extends RuntimeException {

	private static final long serialVersionUID = 4361820653039146950L;

	public UnclonableTypeException() {
        super();
    }

    public UnclonableTypeException(String message) {
        super(message);
    }

    public UnclonableTypeException(String message, Throwable cause) {
        super(message, cause);
    }
	
}

File: Simplex.Core/Exceptions/UnclonableTypeException.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Tmon.Simplex.Exceptions
{
    public class UnclonableTypeException : Exception
    {
        public UnclonableTypeException()
        {
        }

        public UnclonableTypeException(string message)
            : base(message)
        {
        }

        public UnclonableTypeException(string message, Exception innerException)
            : base(message, innerException)
        {
        }
    }
}
